soc/silabs/silabs_siwx91x/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Antmicro
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+rsource "*/Kconfig"
```

Comment:
put inside `if SOC_FAMILY_SILABS_SIWX91X` and move to bottom

Reply:
done. There was also a issue with the way `SOC_FAMILY_SILABS_SIWX91X` was selected.